package.json

```diff
@@ -93,8 +94,8 @@
     "react-native-image-size": "git+https://github.com/Expensify/react-native-image-size#6b5ab5110dc3ed554f8eafbc38d7d87c17147972",
     "react-native-keyboard-spacer": "^0.4.1",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "1.0.10",
-    "react-native-pdf": "^6.2.2",
+    "react-native-onyx": "1.0.12",
+    "react-native-pdf": "^6.6.2",
```

Comment:
Same for this `react-native-pdf` upgrade. Looks like it's trying to sneak in as well